py/kubeflow/testing/run_e2e_workflow.py

```diff
@@ -73,6 +89,15 @@ def __init__(self, name, app_dir, component, job_types, include_dirs, params):
     self.include_dirs = include_dirs
     self.params = params
 
+class WorkflowPyComponent(object):
```

Comment:
Why doesn't PyComponent have include_dirs? We want to apply skipping even in the context of PyFunc.

Reply:
Fixed.